superset/daos/annotation.py

```diff
@@ -30,19 +27,6 @@
 class AnnotationDAO(BaseDAO):
     model_cls = Annotation
 
-    @staticmethod
```

Comment:
Copypasta except the `BaseDAO` is defined as a `classmethod` as opposed to a `staticmethod`.

Reply:
These comments are very helpful while reviewing. Thank you.